index.html

```diff
@@ -165,8 +165,32 @@ <h4 class="connect">
             <li>Frameworks<span class="angle"><img src="images/angle-right.svg" alt=""></span> </li>
             <img src="images/Separator Bottom.svg" alt="" srcset="">
             <li>Skills<span class="angle"><img src="images/angle-right.svg" alt=""></span> </li>
-            <img src="images/Separator Bottom.svg" alt="" srcset="">
+            <img class="last-line" src="images/Separator Bottom.svg" alt="" srcset="">
         </ul>
     </section>
+    <section class="contact-section">
+        <img  class="c-eli1" src="images/contact-Ellipse1.svg" alt="">
+        <img class="c-eli2" src="images/contact-Rectangle1.svg" alt="">
+        <img class="c-eli3" src="images/contact-Ellipse2.svg" alt="">
+        <h3 class="contact-title"> Contact me</h3>
+        <div >
+            <p class="contact-text">If you have an application you are interested in developing, a feature that you need  built or a project that needs coding. 
+                I’d love to help with it!
+            </p> 
+        </div>
+        <form action="https://formspree.io/f/xbjezrzn" method="POST">
+            <div class="fields">
+                <li><input name="name" type="text" maxLength="30" placeholder="John Doe"></li>
+                <li><input type="email" name="email" id="email" placeholder="loremipsum@gmail.com"></li>
+                <li><textarea maxLength="500" name="message" id="" cols="30" rows="10" placeholder="Write your message here..."></textarea></li>
+            </div>
+            <img class="contact-drec" src="images/contact-Rectangle7.svg" alt="">
+            <img class="contact-deli" src="images/contact-Ellipse5.svg" alt="">
+            <button class="contact-button" type="submit"><span class="contact-link">Get in touch</span></button>
```

Comment:
- [ ] Kindly consider adding the button styling interactions following the Figma design. you can achieve it using `:active` and `:focus` pseudoclasses.
![qw](https://user-images.githubusercontent.com/96683843/228310864-91b401ac-a588-4883-aaf7-cf15f4c63e96.jpg)